### DISCUSSION TITLE
fullstack [root@cpzljc dzsyhp]# cargo run --features server    Listening on 127.0.0.1:8080   how to modidy   Listening on 188.188.1.26:8081

### DISCUSSION THREAD

**Discussion by zuijiaoghb** [DioxusLabs/dioxus#2670] [General]
[root@cpzljc dzsyhp]# cargo run --features server
   Compiling serde v1.0.204
   Compiling ahash v0.8.11
   Compiling dioxus-router v0.5.0
   Compiling reqwest v0.12.5
   Compiling dzsyhp v0.1.0 (/opt/dioxus/dzsyhp)
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 45.57s
     Running `target/debug/dzsyhp`
Listening on 127.0.0.1:8080

https://github.com/DioxusLabs/dioxus/tree/main/packages/fullstack/examples/axum-hello-world

**Comment by ealmloff** [comment]
You can set the address with the `addr` function defined in the fullstack config: https://github.com/DioxusLabs/dioxus/blob/v0.5/packages/fullstack/src/config.rs#L54-L60

Or in the git version of dioxus, you can pass the `--addr` argument directly to the binary: `cargo run --features server -- --addr 1.1.1.1 --port 8080`
